src/KOKKOS/compute_reaxff_bonds_kokkos.cpp

```diff
@@ -0,0 +1,192 @@
+// clang-format off
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Richard Berger (LANL)
+------------------------------------------------------------------------- */
+
+#include "compute_reaxff_bonds_kokkos.h"
+#include "atom.h"
+#include "molecule.h"
+#include "update.h"
+#include "force.h"
+#include "memory.h"
+#include "error.h"
+#include "neigh_list.h"
+
+#include "memory_kokkos.h"
+#include "pair_reaxff_kokkos.h"
+#include "reaxff_api.h"
+
+using namespace LAMMPS_NS;
+using namespace ReaxFF;
+
+/* ---------------------------------------------------------------------- */
+
+template<class DeviceType>
+ComputeReaxFFBondsKokkos<DeviceType>::ComputeReaxFFBondsKokkos(LAMMPS *lmp, int narg, char **arg) :
+  ComputeReaxFFBonds(lmp, narg, arg),
+  nbuf(-1), buf(nullptr)
+{
+}
+
+
+/* ---------------------------------------------------------------------- */
+
+template<class DeviceType>
+ComputeReaxFFBondsKokkos<DeviceType>::~ComputeReaxFFBondsKokkos()
+{
+  memoryKK->destroy_kokkos(k_buf, buf);
+}
+
+/* ---------------------------------------------------------------------- */
+
+template<class DeviceType>
+void ComputeReaxFFBondsKokkos<DeviceType>::init()
+{
+  reaxff = dynamic_cast<PairReaxFF*>(force->pair_match("^reax../kk",0));
```

Comment:
I think this may break if using the suffix command--it looks like `force->pair_match` doesn't automatically append the suffix? You can check for `reax` instead, then use the `kokkosable` flag which is set by every pair style.

Reply:
Please see if fd83ed4 addresses your concerns.